lib/helpers/menu_options.rb

```diff
@@ -1,27 +1,41 @@
 require_relative 'object_creation'
+require_relative 'preserve_data'
 
 def list_all(arr, label)
+  data = recover_data(label)
+  data = arr if data.nil?
```

Comment:
I think we should take these out of the ```list_all``` function, we can have a loader class or function to load preserved data when instantiating the App class, this way we won't have side effects on this function, If there's a reason on having it here, please let me know!

Reply:
@Trictonicmp Yeah I just add it for fast, because I was working just with two classes (music and genre), but in the end I think is better to create an updated data method for all the preserved data. I will change it for a method to load